src/Layers/FeatureLayer/FeatureManager.js

```diff
@@ -133,6 +133,8 @@ export var FeatureManager = FeatureGrid.extend({
   _requestFeatures: function (bounds, coords, callback) {
     this._activeRequests++;
 
+    const originalWhere = this.options.where;
```

Comment:
```suggestion
    var originalWhere = this.options.where;
```

looks like the uglify plugin is not liking the es6 `const`